.github/workflows/build.yml

```diff
@@ -24,47 +20,34 @@ jobs:
         uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
         with:
           fetch-depth: 0
-
       - uses: actions/setup-go@3041bf56c941b39c61721a86cd11f3bb1338122a # v5.2.0
         with:
           go-version: "1.23"
-
       - name: Set up Helm
         uses: azure/setup-helm@fe7b79cd5ee1e45176fcad797de68ecaf3ca4814 # v4.2.0
         with:
           version: v3.6.2
-
       - name: build rook
         run: |
           GOPATH=$(go env GOPATH) make clean && make -j$nproc IMAGES='ceph' BUILD_CONTAINER_IMAGE=false build
-
       - name: validate build
         run: tests/scripts/validate_modified_files.sh build
-
       - name: run codegen
         run: GOPATH=$(go env GOPATH) make codegen
-
       - name: validate codegen
         run: tests/scripts/validate_modified_files.sh codegen
-
       - name: run mod check
         run: GOPATH=$(go env GOPATH) make -j $(nproc) mod.check
-
       - name: validate modcheck
         run: tests/scripts/validate_modified_files.sh modcheck
-
       - name: run crds-gen
```

Comment:
We do actually want these empty lines between job steps to help follow them better visually. Please don't use `yq` on the yaml files. This is making the PR huge and also out of scope for the stated intent of the PR, which is some simple make/ci refactoring.

Reply:
@BlaineEXE : now I am unsure how to proceed. Upon @travisn's request, I used a tool to "fix" yamls and came up with `yq`. But I do see that it does changes that are too big.
 
Should I manually revert the unwanted changes? Or is there a better tool?

I don't think dropping the yaml fixes entirely from this PR is an option as the ci will fail.